PWGCF/MultiparticleCorrelations/Tasks/threeParticleCorrelations.cxx

```diff
@@ -83,7 +83,7 @@ struct ThreeParticleCorrelations {
   // Table aliases - Data
   using MyFilteredCollisions = soa::Filtered<soa::Join<aod::Collisions, aod::CentFT0Cs, aod::EvSels>>;
   using MyFilteredCollision = MyFilteredCollisions::iterator;
-  using MyFilteredV0s = soa::Filtered<aod::V0Datas>;
+  using MyFilteredV0s = aod::V0Datas;
```

Comment:
Would it not be best to just replace `MyFilteredV0s` in the subscription by `aod::V0Datas`? As is, having the word `Filtered` there could be confusing... and one could also remove the commented out `Filter`s there as well I think.